package.json

```diff
@@ -21,7 +21,7 @@
   },
   "dependencies": {
```

Comment:
Adding a license should not warrant a change for the package.json file. Why were these upgrades made @oneonone97 ?